ansible/roles_ocp_workloads/ocp4_workload_5gran_deployments_lab/tasks/pre_workload.yml

```diff
@@ -198,6 +243,13 @@
   ansible.builtin.shell:
     cmd: update-ca-trust
 
+- name: Ensure firewalld is disabled
```

Comment:
May I why this workload need disable firewall?

Reply:
We are creating KVM VMs and if it is enabled it does not allow the VMs to access the registry, DNS and IPMI interfaces (sushy) running in the hypervisor.

Reply:
for what instances are you disabling firewalld?

Could you allow the traffic instead of disabling everything? Are those VM exposed in a way?

Reply:
> for what instances are you disabling firewalld?

The baremetal server hosting the VMs 

> Could you allow the traffic instead of disabling everything? Are those VM exposed in a way?

I can take a look, but I think by default it is already stopped in equinox. I include that task just in case in the future is enabled and for my testing on my lab. No, VMs are not exposed are under a virtual NATed network.

Reply:
@hgeaydem  any opinion on this?

Reply:
Just FYI, the equinix base environment we were using didn't have firewalld/iptables installed. So this change does not have any impact on rhpds environments. This is targeting other envs outside RHPDS since we plan to use the playbook outside rhpds as well...

Reply:
sorry we don't mean to block your work, but this catches the attention.

If the firewalld is not there, then don't disable it.

If it''s there in another context, the firewall is probably there for a reason and you should consider allowing the specific rules instead of disabling it entirely?

Reply:
Ok, I commented out the firewalld task so we can continue testing the lab. Meanwhile we are going to take a look at the network flows that need to be enabled. Makes sense? @fridim